sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md

```diff
@@ -68,6 +68,64 @@ title: QuestionAnsweringAuthoringClient
 
 ## Customizations
 
+### General customizations
+
+```yaml
+# Define HTTP 200 responses for LROs to document result model.
+directive:
+- where-operation: QuestionAnsweringProjects_DeleteProject
+  transform: |
+    $.responses["200"] = {
+      description: "Project delete job status.",
+      schema: {
+        "$ref": "#/definitions/JobState"
+      }
+    };
+- where-operation: QuestionAnsweringProjects_DeployProject
```

Comment:
I don't think that's correct - at least, not as authored. Maybe as intended. Progression:

1. https://github.com/Azure/azure-rest-api-specs/blob/8ff0e3b8dc12cd793f4f2208d76f9f3a7f51176c/specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json#L443
2. https://github.com/Azure/azure-rest-api-specs/blob/8ff0e3b8dc12cd793f4f2208d76f9f3a7f51176c/specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json#L488
3. https://github.com/Azure/azure-rest-api-specs/blob/8ff0e3b8dc12cd793f4f2208d76f9f3a7f51176c/specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json#L517

Reply:
Or, were you saying - and I completely misunderstood - that the Operation-Location URI itself is returning a final response that itself points back to the `get` of this and other operations?

Reply:
Not Operation-Location, but our poller impl does an extra GET on the original Uri (used in the PUT) once the operation succeeds. Here's an example in recording:

PUT: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/recordings/test_create_and_deploy_project.pyTestCreateAndDeploytest_deploy_project.json#L209

GET (succeeded): https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/recordings/test_create_and_deploy_project.pyTestCreateAndDeploytest_deploy_project.json#L263

extra GET on the original Uri: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/recordings/test_create_and_deploy_project.pyTestCreateAndDeploytest_deploy_project.json#L292

Reply:
Thanks. I get it now. After talking with @annelo-msft it seems .NET does this as well. /cc @AlexanderSher if that's incorrect, but I'll update my transforms as well. Thanks for catching this and clarifying, and sorry I missed that initially!